tests/packages/test-setuptools/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ['setuptools >= 40.8.0', 'wheel']
```

Comment:
This should be `>= 42` too, `wheel >= 0.36.0`.

Reply:
Agreed. <42 does not work properly with pyproject.toml. I had a package that was broken only on my system but not CI. Took a long time to finally discover it was using a cached setuptools that was buggy under pyproject.toml, it was either 40.x or 41.x.